docs/source/index.md

```diff
@@ -10,6 +10,19 @@ in JupyterLab and the Jupyter Notebook. More specifically, Jupyter AI offers:
 * Support for a wide range of generative model providers and models
   (AI21, Anthropic, Cohere, Hugging Face, OpenAI, SageMaker, etc.).
 
+## JupyterLab support
+
+**Each major version of Jupyter AI supports *only one* major version of JupyterLab.** Jupyter AI 1.x supports
+JupyterLab 3.x, and Jupyter AI 2.x supports JupyterLab 4.x. The feature sets of versions 1.0.0 and 2.0.0
+are the same. We will maintain support for JupyterLab 3 for as long as it remains maintained; that is,
+until JupyterLab 5.0.0 is available.
```

Comment:
```suggestion
are the same. We will maintain support for JupyterLab 3 for as long as it remains maintained.
```

JupyterLab 3 being maintained until the release of JupyterLab 5 isn't a commitment that we can make; perhaps JupyterLab 3 gets deprecated before JupyterLab 5. Regardless, no need to elaborate on how long JupyterLab 3 might get maintained. The general message we want to send is "migrate now" rather than "migrate later".